Week1/homework/app.js

```diff
@@ -1,11 +1,150 @@
 'use strict';
```

Comment:
In general I see that Prettier/automatic code formatting was not yet setup correctly on your computer. Please try to set this up as listed in the instructions here: https://github.com/HackYourFuture/JavaScript2/blob/master/Week1/README.md#software-installation-for-the-javascript-2-module 
If you struggle with this please ask a fellow student or one of the teachers to help this Sunday!

Reply:
Done